usaspending_api/common/helpers/generic_helper.py

```diff
@@ -104,9 +105,17 @@ def dates_are_month_bookends(start, end):
     return False
 
 
-def min_and_max_from_date_ranges(filter_time_periods):
-    min_date = min([t.get("start_date", settings.API_MAX_DATE) for t in filter_time_periods])
-    max_date = max([t.get("end_date", settings.API_SEARCH_MIN_DATE) for t in filter_time_periods])
+def min_and_max_from_date_ranges(filter_time_periods: list) -> tuple:
```

Comment:
These changes might be bleeding into https://federal-spending-transparency.atlassian.net/browse/DEV-4097.

If I'm reading the code correctly, a missing start_date will lead to silently using API_SEARCH_MIN_DATE (and similar logic for missing end_date) which is no longer desired behavior.

Reply:
I will undo the changes since I did not notice that ticket and make a note to circle back after both are merged to make sure Postgres and ES implementations have similar functionality.